modules/fedimint-mint-client/src/lib.rs

```diff
@@ -353,6 +358,18 @@ impl MintClientExt for Client {
             }
         }))
     }
+
+    async fn total_notes(&self) -> Amount {
```

Comment:
`total_amount` is what we call the analagous function on `TieredMulti`

Reply:
We should eventually add this to all client modules so we can have a total of all modules.